seacrowd/sea_datasets/phomt/phomt.py

```diff
@@ -0,0 +1,150 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{PhoMT,
+title     = {{PhoMT: A High-Quality and Large-Scale Benchmark Dataset for Vietnamese-English Machine Translation}},
+author    = {Long Doan and Linh The Nguyen and Nguyen Luong Tran and Thai Hoang and Dat Quoc Nguyen},
+booktitle = {Proceedings of the 2021 Conference on Empirical Methods in Natural Language Processing},
+year      = {2021},
+pages     = {4495--4503}
+}
+"""
+
+_DATASETNAME = "phomt"
+
+
+_DESCRIPTION = """\
+PhoMT is a high-quality and large-scale Vietnamese-English parallel dataset of 3.02M sentence pairs, which is 2.9M
+pairs larger than the benchmark Vietnamese-English machine translation corpus IWSLT15. This is the first large-scale
+Vietnamese-English machine translation study.
+"""
+
+_LANGUAGES = ["vie", "eng"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LOCAL = True
+
+_HOMEPAGE = "https://github.com/VinAIResearch/PhoMT"
+
+_LICENSE = Licenses.MIT.value
```

Comment:
Can you link where you found this? I couldn't find this in the paper or dataset repo.

Reply:
Hi @elyanah-aco, the reason why I put `MIT license` is that I checked the "copyright" statement under `https://github.com/VinAIResearch/PhoMT/blob/main/README.md#copyright-c-2021-vinai-research` and the statement aligns with the content of the MIT license.

Reply:
I'll update the datasheet accordingly then. Thanks @yana-xuyan!